cms/static/js/views/group_configuration_item.js

```diff
@@ -1,77 +1,42 @@
+/**
+ * This class defines an controller view for content experiment group configurations.
```

Comment:
Aw shucks :) Thank you for suggesting to add them. I feel hypocritical for not thinking to do so in the first place, since I really appreciate this type of documentation.